### PR TITLE
OKTA-754148 - Updating scripts to include changeDetails info

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Adding `changeDetails` info to Event Types and CSV
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-754148](https://oktainc.atlassian.net/browse/OKTA-754148)

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue [hunk 1]
```diff
@@ -81,9 +81,15 @@
...
+        <code
+          v-if="eventType.schema.changeDetails !== null && (!Array.isArray(eventType.schema.changeDetails) || eventType.schema.changeDetails.length > 0) "
+          :key="tag"
+          class="event-type-tag"
+        >{{ "changeDetails" }}</code>
       </div>
       <div class="event-type-release">
-        Since: <SmartLink :item="{link: '/docs/release-notes/', text: eventType.info.release}" />
+        <!-- Since: <SmartLink :item="{link: '/docs/release-notes/', text: eventType.info.release}" /> -->
+        Since: {{ eventType.info.release }}
```
Comment (emikhasyak-okta) [inline packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue:92]: Is this intended change? We'll have no link just a release date. If so, could you remove a commented smart link?

File: packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue [hunk 2]
```diff
@@ -81,9 +81,15 @@
           :key="tag"
           class="event-type-tag"
         >{{ tag }}</code>
+        <code
+          v-if="eventType.schema.changeDetails !== null && (!Array.isArray(eventType.schema.changeDetails) || eventType.schema.changeDetails.length > 0) "
```
Comment (emikhasyak-okta) [inline packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue:85]: By looking at event-types.json, changeDetails can be an array or null. We want to show new tag when array is not empty. Although your current check works, `!Array.isArray(eventType.schema.changeDetails)` is always false and doesn't add much to verification.
It probably should be when details is not null also details is an array also the array is not empty:
```
v-if="eventType.schema.changeDetails !== null && Array.isArray(eventType.schema.changeDetails) && eventType.schema.changeDetails.length > 0) "
```

File: packages/@okta/vuepress-site/docs/reference/api/event-types/index.md [hunk 3]
```diff
@@ -17,4 +17,10 @@ The following is a full listing of event types used in the [System Log API](/doc
 
 Download a CSV file with all event types: [Okta Event Types](/docs/okta-event-types.csv).
 
+>**Note:** Certain tags on the event type indicate specific behavior of the associated System Log events:<br>
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/event-types/index.md:20]:
```suggestion
>**Note:** Certain tags on the event type indicate the specific behavior of the associated System Log events:<br>
```

File: packages/@okta/vuepress-site/docs/reference/api/event-types/index.md [hunk 4]
```diff
@@ -17,4 +17,10 @@ The following is a full listing of event types used in the [System Log API](/doc
...
+>
+> * `oie-only`: This event type is only available in Okta Identity Engine enabled orgs
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/event-types/index.md:22]:
```suggestion
> * `oie-only`: This event type is only available in Identity Engine enabled orgs
```

File: packages/@okta/vuepress-site/docs/reference/api/event-types/index.md [hunk 5]
```diff
@@ -17,4 +17,10 @@ The following is a full listing of event types used in the [System Log API](/doc
...
+> * `event-hook-eligible`: This event type is eligible to be selected for use with an event hook
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/event-types/index.md:23]:
```suggestion
> * `event-hook-eligible`: This event type is eligible for use with an event hook
```

File: packages/@okta/vuepress-site/docs/reference/api/event-types/index.md [hunk 6]
```diff
@@ -17,4 +17,10 @@ The following is a full listing of event types used in the [System Log API](/doc
...
+> * `changeDetails`: This event type may include the `changeDetails` object on an associated target within it
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/event-types/index.md:24]: This is a bit hard to understand. What does the "it" at the end refer to?

File: packages/@okta/vuepress-site/docs/reference/api/event-types/index.md (hunk 4 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/reference/api/event-types/index.md:22]: This is first mention, do we no longer require the "Okta"?

File: packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue:91]: Yes, the link was not useful and went to top of Release Notes page; the smartlink cut-off a portion of the release date on right and left side. 

Removed the comment text, thanks.

File: packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue]: Thanks, appreciated the feedback. Implemented your update.

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**